apps/api/src/database/schema.prisma

```diff
@@ -387,6 +387,7 @@ model Document {
   fileType             String?
   isDeleted            Boolean           @default(false)
```

Comment:
The following properties will be version specific:

- blobStorageContainer
- blobStoragePath
- name (assuming this is display name)
- status
- redacted
- fileSize
- fileType
- representationId